__tests__/app.test.js

```diff
@@ -179,4 +179,66 @@ describe("/api/articles", () => {
       });
     });
   });
+  describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: responds with a 200 status and an array of comment objects of length 11 for article_id 1", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(11);
+        });
+    });
+    test("GET 200: returns an array of comment objects each with properties comment_id, body, votes, author, article_id, created_at, each of correct type and with article_id equal to the id passed", () => {
+      const article_id = 1;
+      return request(app)
+        .get(`/api/articles/${article_id}/comments`)
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(11);
+          comments.forEach((comment) => {
+            expect(comment.article_id).toBe(article_id);
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
+          });
+        });
+    });
+    test("GET 200: responds with an array of objects ordered by created_at in descending order", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+    test("GET 200: responds with a 200 status and an empty array if article_id exists but has no associated comments", () => {
+      return request(app)
+        .get("/api/articles/2/comments")
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments.length).toBe(0);
+        });
```

Comment:
Excellent TDD to include a test for this 🏆 
Only thing I noticed was that checking the length isn't quite as robust as `expect(comments).toEqual([])` - strings also have length so an empty string would still pass your test as is